Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -139,10 +139,22 @@ void HotkeyScheduler::Run()
 
   while (!m_stop_requested.IsSet())
   {
-    Common::SleepCurrentThread(1000 / 60);
+    const u64 freelook_update_freq = 200;
```

Comment:
I'd move to a constexpr above, together with the hotkeys 60Hz update rate (I've also done the same change, it's nicer and highlights that there shouldn't be any negative consequence in changing them).